packages/esm-patient-forms-app/src/hooks/use-forms.ts

```diff
@@ -8,8 +8,8 @@ import { isFormFullyCached } from '../offline-forms/offline-form-helpers';
 export function useFormEncounters(cachedOfflineFormsOnly = false) {
   return useSWR([formEncounterUrl, cachedOfflineFormsOnly], async () => {
     const res = await openmrsFetch<ListResponse<FormEncounter>>(formEncounterUrl);
-    // Only show published forms
-    const forms = res.data?.results?.filter((form) => form.published) ?? [];
+    // show published forms and hide component forms
+    const forms = res.data?.results?.filter((form) => form.published && !form.name.includes('component')) ?? [];
```

Comment:
![Screenshot 2022-02-15 at 22 46 03](https://user-images.githubusercontent.com/8509731/154137260-9d33c8cb-64cc-4c43-9e57-f462d35599d4.png)

[String.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) makes case-sensitive matches. Won't the check fail for `Component_POC`? 

Perhaps it should be `!form.name.includes('Component_POC')` instead.

Reply:
That will work for the spa environment, the naming convention adopted is to always name the component forms starting with `component` key word. This `!form.name.includes('Component_POC')` won't work for `component_cdm...` as shown here below
![Screenshot 2022-02-16 at 09 29 38](https://user-images.githubusercontent.com/28008754/154212445-452a905c-5e80-4b6d-889a-395aea64fd7d.png)

consequently i have updated for this to work on both ends `!formName.toLowerCase().includes('component')`

Reply:
Nice, how about a one-liner like:
 
`const forms = res.data?.results?.filter((form) => form.published && !/component_/i.test(form.name)) ?? [];`